modules/nf-core/propr/logratio/main.nf

```diff
@@ -0,0 +1,23 @@
+process PROPR_LOGRATIO {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "conda-forge::r-propr=4.2.6"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/r-propr:4.2.6':
+        'biocontainers/r-propr:4.2.6' }"
+
+    input:
+    tuple val(meta), path(count), val(transformation), val(reference), val(alpha)
+
+    output:
+    tuple val(meta), path("*.logratio.tsv"), emit: logratio
```

Comment:
silly thing- but could you align the emits as you'll see in most other modules? Makes things easier to read.